src/qcardia_data/setup/data_setup.py

```diff
@@ -0,0 +1,40 @@
+from pathlib import Path
+
+import yaml
+
+from qcardia_data.setup.reformat import reformat_mm1, reformat_mm2
+from qcardia_data.setup.test_splits import test_subjects_mm1, test_subjects_mm2
+from qcardia_data.utils import data_to_file
+
+
+def setup_cine(data_path: Path, overwrite: bool = False) -> None:
+    """Reformat and save default test subject split file for M&Ms and M&Ms-2 datasets.
+
+    Args:
+        data_path (Path): path to data folder
+        overwrite (bool, optional): Force reformat all datasets, overwriting any
+            previous reformatted versions. Defaults to False.
+    """
+
+    reformat_mm1(data_path, overwrite=overwrite)
+    reformat_mm2(data_path, overwrite=overwrite)
```

Comment:
Not for now, but maybe at some stage we should add the option to use either mm1 or mm2 - i.e. not neccessarily both

Reply:
Just so you don't have to download all the data? I could add checks to skip reformatting if the original data wasn't found, and tell you it wasn't found. I'll do that when checking the LGE data.